Makefile

```diff
@@ -51,28 +45,30 @@ GOMEGA_VERSION := $(shell awk '/github.com\/onsi\/gomega/ {print $$2}' go.mod)
 # Test coverage threshold
 export COVERAGE_MIN ?= 75
 
-LOCAL_OS := $(shell uname)
-ifeq ($(LOCAL_OS),Linux)
-    TARGET_OS ?= linux
-    XARGS_FLAGS="-r"
-else ifeq ($(LOCAL_OS),Darwin)
-    TARGET_OS ?= darwin
-    XARGS_FLAGS=
-else
-    $(error "This system's OS $(LOCAL_OS) isn't recognized/supported")
-endif
+# Image URL to use all building/pushing image targets;
+# Use your own docker registry and image name for dev/test by overridding the IMG and REGISTRY environment variable.
+IMG ?= $(shell cat COMPONENT_NAME 2> /dev/null)
+REGISTRY ?= quay.io/stolostron
+TAG ?= latest
 
-.PHONY: fmt lint test coverage build build-images
+# go-get-tool will 'go install' any package $2 and install it to $1.
```

Comment:
Since this function no longer uses `$(1)`, it seems like it might always install things to `$(PWD)/bin`, not `$1` like this comment says

Reply:
Ha! Good catch! I'll update it.